qiskit/test/mock/utils/__init__.py

```diff
@@ -14,4 +14,4 @@
 Utils for test backends.
 """
 
-from .configurable_backend import ConfigurableFakeBackend
+# from .configurable_backend import ConfigurableFakeBackend
```

Comment:
I commented out this line because this is causing circular import problem. When functions are imported from `qiskit.test.mock.utils`, it imports from`configurable_backend.py` because it's added to `__init__.py` and `configurable_backend.py` imports from fake_backend.py. Previously `fake_backend.py` does not use `qiskit.test.mock.utils` (they are used in `fake_qasm_backend.py` instead). 

```
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-2-6b681dc21c33> in <module>
----> 1 from qiskit.test.mock.fake_backend import FakeBackendV2
      2 from qiskit.test.mock.backends import FakeJakarta, FakeJakartaV2

~/Documents/Github/HJY/qiskit-dev/qiskit-terra/qiskit/test/mock/__init__.py in <module>
     21 """
     22 
---> 23 from .fake_provider import FakeProvider, FakeLegacyProvider, FakeProviderV2
     24 from .fake_provider import FakeProviderFactory
     25 from .fake_backend import FakeBackend, FakeLegacyBackend

~/Documents/Github/HJY/qiskit-dev/qiskit-terra/qiskit/test/mock/fake_provider.py in <module>
     21 from qiskit.providers.exceptions import QiskitBackendNotFoundError
     22 
---> 23 from .backends import *
     24 from .fake_qasm_simulator import FakeQasmSimulator
     25 from .fake_openpulse_2q import FakeOpenPulse2Q

~/Documents/Github/HJY/qiskit-dev/qiskit-terra/qiskit/test/mock/backends/__init__.py in <module>
     17 
     18 # BackendV1 Backends
---> 19 from .almaden import FakeAlmaden
     20 from .armonk import FakeArmonk
     21 from .athens import FakeAthens

~/Documents/Github/HJY/qiskit-dev/qiskit-terra/qiskit/test/mock/backends/almaden/__init__.py in <module>
     13 """Mock almaden backend"""
     14 
---> 15 from .fake_almaden import FakeAlmaden
     16 from .fake_almaden import FakeLegacyAlmaden

~/Documents/Github/HJY/qiskit-dev/qiskit-terra/qiskit/test/mock/backends/almaden/fake_almaden.py in <module>
     16 
     17 import os
---> 18 from qiskit.test.mock import fake_pulse_backend
     19 
     20 

~/Documents/Github/HJY/qiskit-dev/qiskit-terra/qiskit/test/mock/fake_pulse_backend.py in <module>
     16 
     17 from qiskit.providers.models import PulseBackendConfiguration, PulseDefaults
---> 18 from qiskit.test.mock.fake_qasm_backend import FakeQasmBackend, FakeQasmLegacyBackend
     19 from qiskit.exceptions import QiskitError
     20 from qiskit.test.mock.utils.json_decoder import decode_pulse_defaults

~/Documents/Github/HJY/qiskit-dev/qiskit-terra/qiskit/test/mock/fake_qasm_backend.py in <module>
     19 
     20 from qiskit.providers.models import BackendProperties, QasmBackendConfiguration
---> 21 from qiskit.test.mock.fake_backend import FakeBackend, FakeBackendV2, FakeLegacyBackend
     22 from qiskit.test.mock.utils.json_decoder import (
     23     decode_backend_configuration,

~/Documents/Github/HJY/qiskit-dev/qiskit-terra/qiskit/test/mock/fake_backend.py in <module>
     39 from qiskit.exceptions import QiskitError
     40 from qiskit.test.mock import fake_job
---> 41 from qiskit.test.mock.utils.json_decoder import (
     42     decode_backend_configuration,
     43     decode_backend_properties,

~/Documents/Github/HJY/qiskit-dev/qiskit-terra/qiskit/test/mock/utils/__init__.py in <module>
     15 """
     16 
---> 17 from .configurable_backend import ConfigurableFakeBackend

~/Documents/Github/HJY/qiskit-dev/qiskit-terra/qiskit/test/mock/utils/configurable_backend.py in <module>
     28 from qiskit.providers.models.backendproperties import Nduv, Gate
     29 from qiskit.qobj import PulseQobjInstruction
---> 30 from qiskit.test.mock.fake_backend import FakeBackend
     31 
     32 

ImportError: cannot import name 'FakeBackend' from partially initialized module 'qiskit.test.mock.fake_backend' (most likely due to a circular import) (/Users/junye/Documents/Github/HJY/qiskit-dev/qiskit-terra/qiskit/test/mock/fake_backend.py)
```